src/app/components/tx-module/tx-swap/tx-swap.component.ts

```diff
@@ -117,7 +118,7 @@ export class TxSwapComponent extends TxBase implements OnDestroy {
       this.tokenIn = this.data.pool.token.crs;
       this.tokenOut = this.data.pool.token.src;
     } else {
-      const topTokens = new TokensFilter({limit: 2, direction: 'DESC', orderBy: 'DailyPriceChangePercent'});
+      const topTokens = new TokensFilter({limit: 2, direction: 'DESC', orderBy: this.tokenOrderByTypes.DailyPriceChangePercent});
```

Comment:
same comment, can just set type above then remove `this.` here

Reply:
Done